modules/tinymce/src/core/main/ts/DragDropOverrides.ts

```diff
@@ -223,6 +223,8 @@ const drop = (state: Singleton.Value<State>, editor: Editor) => (e: EditorEvent<
             editor._selectionOverrides.hideFakeCaret();
           });
         }
+
+        editor.fire('dragend');
       }
```

Comment:
```suggestion
      }
      
      editor.fire('dragend');
```
This is in the incorrect place from what I can tell, as it should fire if dragging regardless of if the drop target is valid.